pkg/terminal/proxy.go

```diff
@@ -89,16 +89,6 @@ func (p *Proxy) HandleProxy(user *auth.User, w http.ResponseWriter, r *http.Requ
 		return
 	}
 
-	enabledForUser, err := p.checkUserPermissions(user.Token)
-	if err != nil {
-		http.Error(w, "Failed to check workspace operator state. Cause: "+err.Error(), http.StatusInternalServerError)
-		return
-	}
-	if !enabledForUser {
-		http.Error(w, "Terminal is disabled for cluster-admin users.", http.StatusForbidden)
```

Comment:
I wonder if it makes sense to keep logic on proxy side that will check that workspace is run in protected namespace if the current user is cluster admin?
It would help to provide a bit safer solution if cluster-admin created a terminal in non-protected namespace, and then opened terminal via direct URL, like `${OS_CONSOLE_URL}/console/terminal/non-protected-namespace/workpsace-name` (<- I'm not sure it's the right URL for openning terminal in a dedicated tab)

P.S. This may be already discussed on WTO Cabal or in tmp-terminal channel but I haven't reviewed them yet.